src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleProcessManager.cs

```diff
@@ -72,33 +81,36 @@ public bool IsProcessInitialized()
         }
 
         /// <summary>
-        /// Call xUnit.console.exe with the parameters previously specified
+        /// Call vstest.console with the parameters previously specified
         /// </summary>
         public void StartProcess(ConsoleParameters consoleParameters)
         {
-            this.process = new Process();
-            process.StartInfo.FileName = vstestConsolePath;
-            process.StartInfo.Arguments = string.Join(" ", BuildArguments(consoleParameters));
-
-            //process.StartInfo.WorkingDirectory = WorkingDirectory;
-            process.StartInfo.UseShellExecute = false;
-            process.StartInfo.CreateNoWindow = true;
-
-            //process.StartInfo.RedirectStandardOutput = true;
-            //process.StartInfo.RedirectStandardError = true;
+            var info = new ProcessStartInfo(GetConsoleRunner(), string.Join(" ", BuildArguments(consoleParameters)))
+            {
+                UseShellExecute = false,
+                CreateNoWindow = true,
+                RedirectStandardOutput = true,
+                RedirectStandardError = true
+            };
 
-            EqtTrace.Verbose("VsTestCommandLineWrapper: {0} {1}", process.StartInfo.FileName,
-                process.StartInfo.Arguments);
+            EqtTrace.Verbose("VsTestCommandLineWrapper: {0} {1}", info.FileName, info.Arguments);
```

Comment:
>VsTestCommandLineWrapper [](start = 30, length = 24)

Let's fix the logging to "VsTestConsoleProcessManager : Process Start Info : {0} {1}"